src/panels/lovelace/views/hui-view.ts

```diff
@@ -101,9 +91,17 @@ export class HUIView extends LitElement {
   }
 
   protected render(): TemplateResult {
+    console.log("render");
+
     return html`
       <div id="badges"></div>
-      <div id="columns"></div>
+      <lit-grid-layout
+        .dragHandle=${"hui-card-options"}
+        .items=${this._cards}
+        .layout=${this._layout}
+        .rowHeight=${15}
```

Comment:
Assuming this property has type number in `lit-grid-layout`
```suggestion
        rowHeight="15"
```